---
title: "Arizona SARS-CoV-2 Genomes in a Global Context"
authorLinks: "https://www.tgen.org/patients/pathogen-microbiome/"
affiliations: "TGen, ADHS"
date: "December 2021"
dataset: "https://nextstrain.org/community/tgennorth/arizona-covid-19/AZ?animate=2020-03-13,2021-11-09,1,0,15000&d=map&f_Pango_lineage=A.2.5,A.2.5.2,A.5,AY.1,AY.10,AY.100,AY.101,AY.102,AY.103,AY.105,AY.107,AY.109,AY.110,AY.13,AY.14,AY.16,AY.17,AY.19,AY.2,AY.20,AY.21,AY.24,AY.25,AY.26,AY.3,AY.3.1,AY.33,AY.34,AY.35,AY.36,AY.37,AY.38,AY.39,AY.39.1,AY.4,AY.41,AY.42,AY.43,AY.44,AY.45,AY.46,AY.46.1,AY.46.2,AY.46.4,AY.46.6,AY.47,AY.48,AY.5,AY.5.3,AY.50,AY.52,AY.53,AY.54,AY.57,AY.58,AY.61,AY.62,AY.63,AY.64,AY.65,AY.67,AY.7.1,AY.74,AY.75,AY.75.1,AY.81,AY.88,AY.9.1,AY.9.2,AY.91,AY.92,AY.93,AY.94,AY.98,AY.98.1,B.1,B.1.1.519,B.1.1.7,B.1.289,B.1.351,B.1.427,B.1.429,B.1.516,B.1.526,B.1.551,B.1.617.2,P.1,B.1.243&p=full"
abstract: "Maintained by TGen North and ADHS
 
Before the COVID-19 pandemic began, Arizona was one of the few places in the world that had a robust genomic surveillance system in place that could respond immediately to emerging diseases. As part of this surveillance system, the Translational Genomics Research Institute (TGen North) had (and still has) partnerships with the Arizona Department of Health Services (ADHS) and several other public health agencies, including tribal and county health departments, as well as many healthcare facilities around the state. Because of this, TGen North was one of the first laboratories in the U.S. to not only set up a COVID-19 diagnostic testing lab, but to launch SARS-CoV-2 genomic surveillance. We sequenced our first SARS-CoV-2 genome on March 11, 2020, and our first diagnostic test was run on March 16, 2020.
 
Genomic surveillance has shown that today’s globalized society resulted in a homogenous distribution of SARS-CoV-2 lineages: most of the viral lineages found in Arizona are also seen across the world, but a limited number of virus lineages were confined within the state. Importantly, genomic surveillance is also able to illustrate the movement of SARS-CoV-2 from country to country and person to person.
 
Our narrative tells the stories of those virus transmissions and SARS-CoV-2 evolution, set in the context of Arizona’s unique ecology and demography, and the remarkable response of our public health and healthcare workers."
---





<!-- Page 2 -->
# [Arizona’s COVID-19 Timeline](https://nextstrain.org/community/tgennorth/arizona-covid-19/)

Arizona detected and successfully contained its first COVID-19 case on January 26. Early March marked the first recorded cases of community transmission, and by late March state public health officials declared the virus “Widespread”. As in other states, throughout June and July, hospitalizations were at an all-time high, and infection rates skyrocketed.
 
Despite intensive mitigation efforts implemented statewide, Arizona began to experience their second wave beginning in mid-November that has continued into 2021. The Delta variant was introduced to Arizona in late January 2021, and began spreading more widely in May. As of December 2021, Arizona has reported over 1.2 million cases and 21,000 deaths.
 
The Arizona Department of Health Services maintains a [COVID-19 Data Dashboard](https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php), and TGen North developed the [Arizona COVID-19 Sequencing Dashboard](https://pathogen.tgen.org/covidseq-tracker/). From the ~1.2 million positive COVID-19 cases in Arizona, TGen North has sequenced and analyzed >52,000 SARS-CoV-2 genomes.

* [Timeline: WHOs COVID-19 Response](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/interactive-timeline#event-71)
* [ACGU published analysis of early pandemic in Arizona](https://mbio.asm.org/content/11/5/e02107-20)

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-timeline-horizontal.png" width="90%">
```





<!-- Page 3 -->
# [An Early Pandemic Analysis of SARS-CoV-2 Population Structure and Dynamics in Arizona](https://nextstrain.org/community/tgennorth/arizona-covid-19/)

Arizona’s first COVID-19 case was in a 26-year-old symptomatic male who had recently traveled to Wuhan City, China. This case was rapidly quarantined. Genomic analysis of the virus supports the public health contact tracing conclusions that no subsequent transmission occurred.
 
Viral sequencing data was further generated by TGen and other labs, allowing for a comprehensive genomic analysis of SARS-CoV-2 introduction into and spread across AZ. 
 
This analysis revealed that:
 
* SARS-CoV-2 was introduced at least twelve times by April 2020 from other countries and parts of the US.
* The first introductions resulting in community transmission were in mid to late February.
* Introductions mostly resulted from domestic travel.
 
This [early pandemic analysis](https://tgen.us10.list-manage.com/track/click?u=6b0e128788a10661a1fb2d549&id=f3e6925358&e=81c3146932) was supported by: The NARBHA Institute, Flinn Foundation, The Virginia G. Piper Charitable Trust, Blue Cross and Blue Shield of Arizona, the National Institutes of Health, the David and Lucile Packard Foundation, the University of Arizona College of Science and Office of Research Innovation and Impact, and the BIO5 Institute.

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-5.png" width="650">

**First introductions were as early as mid-February**. Molecular clock analyses back-calculate the date that each lineage of virus spreading in Arizona (color-coded above) originated from one individual, i.e., the person that carried the virus into the state. The five major early lineages of virus in Arizona were estimated to be carried in in mid-February to late March.

<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/blank.png" width="100%" height="70">

<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-6.png" width="650">

**First introductions were mostly from domestic travel**. For six of the earliest strain types of SARS-CoV-2 in Arizona (A.1 to B.1.29), those same strain types were found in other parts of North America (blue) and Europe (gray). This demonstrates that the introductions of COVID-19 to Arizona likely came with people traveling within the U.S. or from Europe.
```





<!-- Page 4 -->
# [Outbreak of COVID-19 in Group Home Setting](https://nextstrain.org/community/tgennorth/arizona-covid-19/Hozhoni?c=gt-S_5&d=tree&m=div&p=full)

An outbreak of COVID-19 occurred among staff and residents associated with a multi-residential group home that provides care for adults with intellectual and developmental disabilities. A potential transmission network was developed to illustrate person-place epidemiologic linkages and further demonstrate the connections between staff and residents.
 
The genomic findings paired with the epidemiologic data suggest that infected staff introduced COVID-19 into this setting. Staff initially interacted with residents from multiple homes until COVID-19 was discovered. Enhanced precautions and infection control measures by staff and timely intervention by public health officials likely curbed this outbreak.
 
View the article [here](https://www.frontiersin.org/articles/10.3389/fpubh.2021.668214/full).





<!-- Page 5 -->
# [Household Cluster of SARS-CoV-2 Disease in Cat, Dog, and Pet Owner](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Arizona’s COVID-19 & Pets Program is a surveillance study to characterize how SARS-CoV-2 impacts companion animals living in households with people who have tested positive for the virus. Among the enrolled pets, we identified an infected cat and dog from the same household; both animals were asymptomatic but had close contact with their owner. Whole genome sequencing of animal and owner specimens revealed identical viral genomes, suggesting zoonotic transmission of SARS-CoV-2 from human to at least one pet.
 
This is one of the first studies in the U.S. documenting transmission of COVID-19 from a pet owner to pets.

```auspiceMainDisplayMarkdown
<div> <img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-8.png" width="500">

<!-- <img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/blank.png" width="100%" height="70"> -->

<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-9.png" width="500"> </div>
```





<!-- Page -2 -->
# [Public Health Significance of Genomic Surveillance](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Using genomic epidemiology, we have a better understanding of the circulation of strains of the SARS-CoV-2 virus among Arizona communities, which is critical to guide an effective public health response to track and curb the current and future spread of the virus.
 
We’ve used whole genome sequencing and data analysis to help our public health and healthcare partners.
 
* detect if samples are related or not
* if a cluster of cases is a true outbreak or not
* identify origins of introductions of different COVID-19 lineages to a region
* locate hot spots or identify superspreader events
* study mutations in the viral genome and their potential for causing increased transmission or virulence
* and track variants in communities throughout Arizona

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-10.png" width="650">

An artist rendering of a SARS-CoV-2 virus particle as its spike proteins (red) attach to ACE2 receptors (dark blue) on a human cell.

The ACE-2 receptor on mammalian cells (a physiological trait that many species have in common) serves as a docking station for when the SARS-CoV-2 virus comes calling. Over the course of this pandemic, the virus has acquired mutations in the spike proteins that slightly change the shape of the virus, allowing it to bind with even greater affinity to the ACE-2 receptors on cells. Studies have shown that the SARS-CoV-2 spike proteins fit just as well with mink and feline ACE-2 receptors as they do humans.
```





<!-- Page -1 -->
# [Acknowledgements](https://nextstrain.org/community/tgennorth/arizona-covid-19/)

- TGen
  - Dave Engelthaler
  - Jolene Bowers
  - Darrin Lemmer
  - Megan Folkerts
  - Chris French
  - Hayley Yaglom
  - Danielle Vasquez
  - Ashlyn Pfeiffer
  - Amber Jones
  - Adina Doyle
  - Krystal Sheridan
  - Sara Wilbur
  - Jim Schupp
  - Marjorie Nguyen
  - Heather Centner
  - Amber Whitaker
  - Juan Monroy-Nieto

- NAU
  - Paul Keim
  - Crystal Hepp
  - Jason Ladner
  - Greg Caporaso
  - Jason Sahl
  - Evan Bolyen

- ASU
  - Efrem Lim
  - Brenda Hogue

- UA
  - Brendan Larsen
  - Grace Quirk
  - Thomas Watts
  - Michael Worobey


- ADHS
  - Taylor Martins

## Acknowledgements:

TGen  would like to acknowledge the tireless efforts of Arizona’s local, tribal and state public health officials and healthcare workers, and to thank the other laboratories around the world submitting SARS-CoV-2 genomes to be shared with the scientific community.

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-tgen.png" width="300">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-nau.png" width="300">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-ua.png" width="300">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-asu.png" width="300">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-adhs.png" width="300">
```
