---
title: "AZ-Strain: Genomic Epidemiology of SARS-CoV-2 in Arizona Maintained by the AZ COVID Genomics Union (ACGU)"
authors: "Dave Engelthaler, Jolene Bowers, Darrin Lemmer, Megan Folkerts, Hayley Yaglom, Daneille Vasquez, Ashlyn Pfeiffer, Chris French" 
authorLinks: "https://www.tgen.org/patients/pathogen-microbiome/"
affiliations: "ACGU, TGen"
date: "2020 Sep 14"
dataset: "https://nextstrain.org/community/tgennorth/arizona-covid-19/7-13-20?d=map"
abstract: "Faculty at the Translational Genomics Research Institute (TGen), an affiliate of City of Hope, Northern Arizona University (NAU), University of Arizona (UArizona) and Arizona State University (ASU) launched the Arizona COVID Genomics Union (ACGU) to collaboratively track SARS-CoV-2, and examine how it evolves and how it spreads into, within and out of Arizona.
 
The ACGU uses state-of-the-art sequencers, custom computational analysis workflows, and supercomputers to determine the sequence of the virus’s RNA genome and reconstruct its evolutionary history with phylogenetics, to track its movements. TGen has so far sequenced SARS-CoV-2 genomes from over 3,000 COVID-19 positive samples for the ACGU, and additional sequencing was performed at ASU and UArizona, from among the more than 200,000 positive cases in Arizona, making it one of the most robust such efforts in the nation. ACGU receives Arizona samples collected by state, county, tribal and private healthcare systems.
 
ACGU scientists take advantage of small changes or mutations in the virus’ genome, which naturally occur over time as the virus reproduces, to track the spread of the virus. By comparing mutations observed in Arizona to those present in strains circulating across the globe, they can determine when and from where the virus was introduced to Arizona, and comparing strains within Arizona they can map its spread within and among Arizona’s communities."
---

# [SARS-CoV-2 in Arizona](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

COVID-19 has gravely impacted Arizona, sweeping through urban areas, tribal communities, correctional facilities, assisted living facilities, and more over the past nine months. Arizona detected its first COVID-19 case on January 26th, but it was not until late March that state public health officials reported widespread community transmission. In June and July, Arizona’s peak during this pandemic, hospitalizations were at an all-time high, and rates among individuals aged 20-44 skyrocketed. Arizona’s case rate per 100,000 was almost double what was reported for the United States. As of September 15, 2020, Arizona has reported over 209,200 cases and 5,340 deaths. The Arizona Department of Health Services shares a [COVID-19 Data Dashboard](https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php) updated daily.

```auspiceMainDisplayMarkdown
# Arizona COVID-19 Timeline

![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-timeline.png)
```

# [Arizona SARS-CoV-2 Genomes in a Global Context](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst?d=tree&f_division=Arizona&p=full)

This build includes a dataset of 812 SARS-CoV-2 genomes sequenced from COVID-19 positive patients from across Arizona. Testing was conducted at TGen North Clinical Laboratory, Arizona State Public Health Laboratory, and Sonora Quest Laboratory between March XX and June XX.
 
The Arizona genomes are shown here with a global dataset of approximately XX genomes shared in the GSAID database. A majority of the Arizona samples fall within the B1 lineage which is the dominating North American strain.

# [Arizona’s First COVID-19 Case was Successfully Contained](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst?d=tree&f_division=Arizona&p=full&s=TG268188::Coconino::B.2::0.99::20-5463596::2020-03-17)

COVID-19 hit AZ on January 22, 2020, when a 26-year-old male affiliated with a local university presented to a university healthcare clinic in Maricopa County, Arizona reporting a 2-day history of nonproductive cough. He denied fever and other symptoms at presentation. He had returned from travel to China, including Wuhan City, 3 days prior to presentation, and reported having contact with one person with cough and fever while in China. The case was rapidly quarantined. Despite his specimens testing positive at  the CDC 18 days after diagnosis, no SARS-CoV-2 transmission was detected among 10 high-risk contacts, including 1 intimate contact, based on serial SARS-CoV-2 testing. The US CDC published a genome sequence for the sample collected from [this initial case](https://www.ncbi.nlm.nih.gov/nuccore/MN997409.1). 
 
Genomic analysis by the ACGU supports the public health contact tracing conclusions; continued genomic surveillance shows no evidence that this first case resulted in cryptic (undetected) transmission, as no strains of virus collected later in Arizona descended from this lineage. 
 
The AZ1 genome contained one derived substitution (C to T at nucleotide position 29,031) that we did not observe in any other Arizona sequences. This substitution was, however, shared with 12 other genomes included in our analysis, all of which were sampled in China or Japan. This is consistent with infection of AZ1 occurring during documented travel to China.
 
**_Rapid public health response efforts, including quarantining, testing, and contact tracing, prevented this case from potentially igniting a much earlier outbreak in AZ._**

# [Evidence of Multiple Introductions of SARS-CoV-2 into Arizona in February and March, Mostly from Domestic Travel](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

SARS-CoV-2 was introduced into AZ at least 12 times by early April
Multiple distinct SARS-CoV-2 lineages co-circulated in AZ; genomic analyses highlight 11 distinct lineages and/or sublineages in Arizona by early April  None of these lineages were unique to Arizona; they have all been documented in other parts of the United States and, in most cases, also in multiple countries around the world. In fact, the geographic ubiquity of the major SARS-CoV-2 lineages, along with the common observation of identical virus genomes sampled in multiple U.S. states, as well as other countries and continents, clearly demonstrates how frequently this virus has been moved among locales. Given the timing and size of the Arizona outbreak in March, relative to outbreaks in other locations, it is unlikely that any of these 11 lineages arose within Arizona. Therefore, the number of observed distinct lineages (12, including AZ1) represents a conservative estimate for the number of independent introductions of SARS-CoV-2 into Arizona.

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-1.png)

### SARS-CoV-2 was introduced into AZ at least 12 times by early April at least 12 times by early April
Multiple distinct SARS-CoV-2 lineages co-circulated in AZ; genomic analyses highlight 11 distinct lineages and/or sublineages in Arizona by early April  None of these lineages were unique to Arizona; they have all been documented in other parts of the United States and, in most cases, also in multiple countries around the world. In fact, the geographic ubiquity of the major SARS-CoV-2 lineages, along with the common observation of identical virus genomes sampled in multiple U.S. states, as well as other countries and continents, clearly demonstrates how frequently this virus has been moved among locales. Given the timing and size of the Arizona outbreak in March, relative to outbreaks in other locations, it is unlikely that any of these 11 lineages arose within Arizona. Therefore, the number of observed distinct lineages (12, including AZ1) represents a conservative estimate for the number of independent introductions of SARS-CoV-2 into Arizona.
 
### SARS-CoV-2 was introduced into AZ in mid to late February
Molecular clock analyses showed the introductions of SARS-CoV-2 that resulted in community transmission in AZ were in February and March. 
 
XX% of our sequences descend from strains that were initially circulating widely in Europe but have since dominated the outbreak in the United States. In addition, we show that the first reported case of community transmission in Arizona descended from the Washington state outbreak that was discovered in late February. Notably, none of the observed transmission clusters are epidemiologically linked to the original travel-related case in the state, suggesting successful early isolation and quarantine.

### Introductions likely mostly resulted from domestic travel
The first reported case of “community” transmission occurred in Arizona in early March descended from the Washington state outbreak that was discovered in February. More than 80% of the SARS-CoV-2 genome sequences from Arizona COVID-19 cases descended from at least 11 separate lineages that were initially circulating widely in Europe, and by travel have since dominated the outbreak throughout the U.S. None of the observed transmission clusters are epidemiologically linked to the original travel-related case in Arizona, suggesting successful early isolation and quarantine.

```

# [COVID-19 Level in AZ is “Widespread”](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

On March 26th, the Arizona Department of Health Services updated the community transmission level of COVID-19 in Arizona to widespread, indicating that 12 or more of the fifteen counties throughout the state had confirmed COVID-19 cases. 
 
The Nextstrain dataset shows XXX genomes from XX counties reporting COVID-19 cases from up to March 26th. Colors indicate the strain of virus, indicating that multiple lineages spread among several counties within AZ.


```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-2.png)
```

# [An Early Pandemic Analysis of SARS-CoV-2 Population Structure and Dynamics in Arizona](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Initial findings reported by the Arizona COVID-19 Genomics Union (ACGU) suggest that the state experienced no new COVID-19 cases after the first reported case in late January, until at least 11 distinct incursions occurred between mid-February and early April. The published results appear in the scientific journal mBio. This study was supported by: The NARBHA Institute, Flinn Foundation, The Virginia G. Piper Charitable Trust, Blue Cross and Blue Shield of Arizona, the National Institutes of Health, the David and Lucile Packard Foundation, the University of Arizona College of Science and Office of Research Innovation and Impact, and BIO5 Institute.

# [Outbreak of COVID-19 in a Linked Group Home Setting](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-3.png)
```

# [Outbreak Story 2](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
```

# [Outbreak Story 3](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-4.png)
```

# [Rapid Transmission of SARS-CoV-2 in Northern Arizona Tribal Community with Low Mortality](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst?c=gt-S_245&m=div&p=full)

- Rapid Transmission of SARS-CoV-2 in Northern Arizona Tribal Community with Low Mortality

# [Summary](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Return to timeline - highlight uniqueness of AZ epi circumstances - anti-seasonal effects as people congregate indoors in summer.
 
Our data tell us that the initial Arizona case with travel to China in January did not lead to sustained local transmission, and that multiple introductions (>12) of the virus occurred from multiple countries and states, some of which contributed to significant transmission in Arizona. Mutations have been found that may lead to increased transmission and decreased virulence.


# [Public Health Significance](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Using genomic epidemiology, we have a better understanding of the circulation of strains of the SARS-CoV-2 virus among Arizona communities, which is critical to guide an effective public health response to curb the spread of the virus now and in the future. Whole genome sequencing of the viral genome from infected persons’ clinical specimens, along with information about the collection of the specimens (when and where) allows for phylogenetic (ie, family tree) reconstruction to track the circulation and transmission of the virus locally and globally. We use this data to understand viral movement, identify origins of introductions of different COVID strains to the region, locate hot spots of local transmission and clusters of disease, and possibly determine the source of infection. We can also study mutations in the viral genome,  and their potential for causing increased transmission or virulence.

# [Methodology](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

```auspiceMainDisplayMarkdown
```

# [Acknowledgements](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-tgen.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-nau.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-pmi.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-ua.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-asu.png)
```


