---
title: "TGen North SARS-CoV-2"
authors: "Dave Engelthaler, Jolene Bowers, Darrin Lemmer, Megan Folkerts, Hayley Yaglom, Daneille Vasquez, Ashlyn Pfeiffer, Chris French" 
authorLinks: "https://www.tgen.org/patients/pathogen-microbiome/"
affiliations: "ACGU, TGen"
date: "2020 Sep 14"
dataset: "https://nextstrain.org/community/tgennorth/arizona-covid-19?d=map"
abstract: "COVID-19 has gravely impacted Arizona, sweeping through urban areas, tribal communities, correctional facilities, assisted living facilities, and more over the past nine months. Arizona detected its first COVID-19 case on January 26th, but it was not until late March that state public health officials reported widespread community transmission. In June and July, Arizona’s peak during this pandemic, hospitalizations were at an all-time high, and rates among individuals aged 20-44 skyrocketed. Arizona’s case rate per 100,000 almost doubled what was reported for the United States. As of September 15, 2020, Arizona has reported over 209,200 cases and 5,340 deaths. The Arizona Department of Health Services shares a COVID-19 Data Dashboard updated daily.

Sixty-five percent of the state’s 7.2 M population lives in the Phoenix Metropolitan Area and its surrounding communities. To understand the introduction and spread of SARS-CoV-2 in this region, the Translational Genomics Research Institute (TGen North), Northern Arizona University, University of Arizona, and their partners are performing viral sequencing and genomic epidemiologic analysis of positive COVID-19 case samples."
---

# [Genomic Epi](https://nextstrain.org/community/tgennorth/arizona-covid-19)

This build includes a dataset of XX SARS-CoV-2 genomes sequenced from COVID-19 positive patients from across Arizona. All 
 
In this report, we describe insights from a recently updated dataset of 772 SARS-CoV-2 genomes sequenced from COVID-19 patients primarily from the Boston area in Massachusetts, USA. All 772 genomes were obtained from nasopharyngeal swabs from individuals with confirmed SARS-CoV-2 infection collected by the Massachusetts Department of Public Health between March 3, 2020 and April 18, 2020, and by Massachusetts General Hospital between March 4, 2020 and May 9, 2020. 

# [Arizona’s First COVID-19 Case](https://nextstrain.org/community/tgennorth/arizona-covid-19)

On January 22, 2020 a 26-year-old male affiliated with a local university presented to a university healthcare clinic in Maricopa County, Arizona reporting a 2-day history of nonproductive cough. He denied fever and other symptoms at presentation. He had returned from travel to China, including Wuhan City, 3 days prior to presentation, and reported having contact with one person with cough and fever while in China. Despite his specimens testing positive 18 days after diagnosis, no SARS-CoV-2 transmission occurred among 10 high-risk contacts, including 1 intimate contact, based on serial SARS-CoV-2 testing. This was the first mildly ill, non-hospitalized U.S. COVID-19 case. Arizona did not have another confirmed COVID-19 case until March 2, 2020.

```auspiceMainDisplayMarkdown
Something about public health response efforts - quarantine - likely played role in curbing spread

Continued genomic surveillance shows no evidence that this case resulted in cryptic (undetected) transmission, as no other AZ case samples are closely related to this one.
```

# [Multiple introductions](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Phylogenetic analysis of XX genomes from across Arizona revealed at least XX distinct introductions inferred to have occurred during February and March. XX% of our sequences descend from strains that were initially circulating widely in Europe but have since dominated the outbreak in the United States. In addition, we show that the first reported case of community transmission in Arizona descended from the Washington state outbreak that was discovered in late February. Notably, none of the observed transmission clusters are epidemiologically linked to the original travel-related case in the state, suggesting successful early isolation and quarantine.

# [Origins of introductions?](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Origins appear mostly from domestic travel

```auspiceMainDisplayMarkdown
* Add figures from paper?
```

# [Widespread Community Transmission](https://nextstrain.org/community/tgennorth/arizona-covid-19)

highlight a date range where widespread designation claimed
Map of AZ with cases in as many counties as we have data for.
 
The Arizona Department of Health Services (ADHS) has updated the community transmission level of COVID-19 in Arizona to widespread. Widespread transmission indicates that cases have been confirmed in twelve or more counties throughout the state. As of today, Arizona has confirmed 508 cases in 13 counties. There have been 8 deaths reported due to COVID-19.


# [Superspreader events?](https://nextstrain.org/community/tgennorth/arizona-covid-19)

```auspiceMainDisplayMarkdown
* TBD w new data
```

# [Cluster/Outbreak Story 1](https://nextstrain.org/community/tgennorth/arizona-covid-19)

```auspiceMainDisplayMarkdown
* Coconino Group Home
```

# [Cluster/Outbreak Story 2](https://nextstrain.org/community/tgennorth/arizona-covid-19)

```auspiceMainDisplayMarkdown
* Mohave Cluster? See image, we need to get epi info
```

# [Mutations/Transmission in Tribal Community](https://nextstrain.org/community/tgennorth/arizona-covid-19)

```auspiceMainDisplayMarkdown
* WMA Story / H245Y (T67I) and R209I mutations
```

# [Summary](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Return to timeline - highlight uniqueness of AZ epi circumstances - anti-seasonal effects as people congregate indoors in summer.
 
Our data tell us that the initial Arizona case with travel to China in January did not lead to sustained local transmission, and that multiple introductions (>12) of the virus occurred from multiple countries and states, some of which contributed to significant transmission in Arizona. Mutations have been found that may lead to increased transmission and decreased virulence. 

# [Public Health Significance??](https://nextstrain.org/community/tgennorth/arizona-covid-19)

Using genomic epidemiology, we have a better understanding of the circulation of strains of the SARS-CoV-2 virus among Arizona communities, which is critical to guide an effective public health response to curb the spread of the virus now and in the future. Whole genome sequencing of the viral genome from infected persons’ clinical specimens, along with information about the collection of the specimens (when and where) allows for phylogenetic (ie, family tree) reconstruction to track the circulation and transmission of the virus locally and globally. We use this data to understand viral movement, identify origins of introductions of different COVID strains to the region, locate hot spots of local transmission and clusters of disease, and possibly determine the source of infection. We can also study mutations in the viral genome,  and their potential for causing increased transmission or virulence.
