---
title: "AZ-Strain: Genomic Epidemiology of SARS-CoV-2 in Arizona"
authorLinks: "https://www.tgen.org/patients/pathogen-microbiome/"
affiliations: "ACGU, TGen"
date: "2020 Sep 14"
dataset: "https://nextstrain.org/community/tgennorth/arizona-covid-19/7-13-20?d=map"
abstract: "An Overview of Genomic Epidemiology of SARS-CoV-2 in Arizona

Like many states, Arizona has reported an incredible number of COVID-19 cases, more than 250,000, since the beginning of this pandemic. But here in Arizona, we have sequenced and analyzed over 5,000 SARS-CoV-2 genomes, making this sequencing effort one of the most extensive in the nation. These efforts help us to map the transmission of the virus within and among Arizona communities, and study mutations in the viral genome to understand their potential for causing increased transmission or virulence. This information can alert public health to superspreader events, hot spots of local spread and clusters of disease, as well as aid in the identification of new introductions of different COVID-19 strains to the region, and possibly determine the source of infection."
---

# [Arizona SARS-CoV-2 Genomes in a Global Context](https://nextstrain.org/community/tgennorth/arizona-covid-19/?c=location&p=full&d=tree)

By comparing Arizona’s virus genomes to those from around the world, we have learned a lot about SARS-CoV-2 in Arizona. First, COVID-19 came into Arizona from several different places, as demonstrated by this phylogenetic tree that maps how the different strains of the virus from around the world are related, much like a family tree. The colors represent the fifteen counties in Arizona. The strains that are now circulating are closely related to strains from many other countries; but, the vast majority are shared with other U.S. states or Europe. By analyzing our phylogenetic trees closely, we specifically know that the virus was introduced at least twelve times during the early part of the pandemic, and that the first introductions resulting in community transmission in Arizona were in mid to late February, and mostly from domestic travel.

# [Arizona’s First COVID-19 Cases were Successfully Contained](https://nextstrain.org/community/tgennorth/arizona-covid-19/AZ1?c=location&p=full&s=USA-AZ1-2020)

ne of the very first cases of COVID-19 in the U.S. was identified in Arizona on January 22, when a 26-year-old male presented to a health clinic, reporting a two-day history of nonproductive cough. He recently returned from Wuhan City, China and reported having contact with a person with cough and fever while traveling. Rapid public health response efforts, including quarantining, testing, and contact tracing really helped to prevent further transmission of this initial case. Genomic analysis shows this virus strain is closely related to others from Asia - pointing to its origin - and most importantly, supports the public health conclusions that spread was stopped, as we don’t see any more viruses from Arizona related to this one. This is one example of how sequencing can be used in tandem with contact tracing and epidemiologic information to illustrate the success of public health actions.

# [COVID-19 Level in Arizona is “Widespread”](https://nextstrain.org/community/tgennorth/arizona-covid-19/?c=location&d=map&f_division=Arizona&p=full&r=location)

On March 26, just 3 weeks after community transmission had been identified by public health in Arizona, the Arizona Department of Health Services declared COVID-19 in Arizona to be “Widespread”, indicating that ≥12 of the 15 counties reported cases. This map shows that we sequenced 64 genomes from COVID-19 positive cases from all 12 of those counties by that date, allowing us to add genomic information to further tell the story and retrospectively assess the impact of early public health efforts implemented.

# [Genomic Epidemiology of a COVID-19 Outbreak Among Residents and Staff Linked to a Developmentally Disabled Adult Group Home](https://nextstrain.org/community/tgennorth/arizona-covid-19/Hozhoni?c=gt-S_5&d=tree&m=div&p=full)

Now, I am going to dive into a few Arizona stories that demonstrate how rapid genomic analyses can provide a better understanding of pathogen transmission patterns and help guide public health response efforts.
 
In mid-May, local public health officials responded to an outbreak of COVID-19 among staff and residents associated with a multi-residential group home that provides care for adults with intellectual and developmental disabilities. Samples were collected at 16 of the homes. In four of the homes all the residents tested positive, and in the remaining 12 houses where samples were collected, all residents tested negative. 
 
All genomes we sequenced from this outbreak are closely related to each other, which you can see here by the very few numbers of mutations separating these samples. The epidemiologic and genomic evidence correlate and suggest that an infected staff member introduced COVID-19 into this setting. Staff initially interacted with residents from multiple homes until COVID-19 was discovered, when staff/management responded by restricting travel between homes. Through our sequencing efforts of samples collected from positive COVID-19 cases in the same region of Arizona, we also learned that this outbreak was not solely confined to the group home and that some community transmission was involved. The enhanced precautions and infection control measures taken by staff and timely intervention by public health officials likely curbed this outbreak.

# [Rapid Transmission of SARS-CoV-2 in an Arizona Tribal Community with Low Mortality](https://nextstrain.org/community/tgennorth/arizona-covid-19?c=mutation&d=tree&p=full)

We have been working closely with one of Arizona’s Native American communities that was hit hard by COVID-19. This tribal community has experienced high infection rates, but very low mortality likely driven by successful on the ground interventions from public health staff and community health workers including, rapid contact tracing, testing, and early clinical management. Early on, evidence pointed to four independent clusters of COVID-19 occurring in the tribal community, however genomic analyses show that cases associated with these clusters stemmed from a single transmission chain. The frequent interactions amongst community contacts and family members living in close proximity to each other, and NOT traveling off the reservation, likely played a role in the widespread transmission of the virus observed in this community. This information is crucial for understanding that continuous travel off the reservation was not responsible for spreading COVID-19 into this community and reinforces the imperative of working with the tribal community to interrupt transmission.
 
The tribe’s index case (sample collected March 29) was a healthcare employee that had an onset of fever and cough on March 25, and a history of recent travel to Phoenix, but continued to work. Genomic analysis shows that this case seeded the outbreak comprising the majority of cases on this reservation, illustrated by the monophyletic clade defined by an amino acid change in the spike protein H245Y. All identified cases in April and most in May from this community make up this clade, which demonstrates the rapid spread of a single clone. We do not know at this time if this would result in any phenotypic change.

# [Contact Tracing Identified Likely Transmission Pattern of SARS-CoV-2 from Index Case](https://nextstrain.org/community/tgennorth/arizona-covid-19)

The index case had five household close contacts, all of whom became symptomatic and tested positive. Given the high attack rate observed, extensive contact tracing was conducted. Two coworkers developed mild symptoms and one tested positive. Attempting to prevent spread to the children in that household, four asymptomatic children were sent to live in another home. The children and members of the second household subsequently tested positive, providing evidence that transmission in the household had already occurred. This turned out to be a common theme among the neighborhoods in this community, and provided valuable insight to public health about the uniqueness of tribal community members’ response to a spreading disease.
 
The phylogenetic analysis of this cluster correlates well to the series of events uncovered in the epidemiological investigation, and suggests successful isolation efforts with no further spread - in and among these households. However, the larger outbreak in this tribal community stemmed from this single case, and this was uncovered only by genomic analysis. This indicates that, despite rapid, efficient contact tracing and isolation directives from public health officials, transmission occurred that was not curbed. One limitation is that our genomic data does not capture all the cases seen in this community, therefore I want to emphasize here the importance of looking at genomic and epidemiologic data together.

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-index.png" width="80%">
```

# [Public Health Significance](https://nextstrain.org/community/tgennorth/arizona-covid-19?d=map&d=tree)

The application of genomic epidemiology into public health practice is growing, and has tremendous benefits as highlighted with these Arizona stories. Although we can tell so much more of the epidemiologic story with genomics, a good understanding of the limitations is important, just like it is for the shoe-leather findings. First, as I indicated with the previous example, we need to remember that we are only subsampling the virus population, and there are likely other cases we don’t see in our genomic analyses that might play an important role in transmission. This can affect the interpretations that we are able to make with the data. Second, just because two samples are identical doesn’t mean that there weren’t intermediate transmission events (i.e. other people) connecting two cases. It is critical to take genomic and epidemiologic information, along with additional data elements, in the context of the other when interpreting findings, formalizing conclusions, and implementing actions.
 
As a former public health epidemiologist, I have learned the value of these new technologies and how they can significantly advance the state of infectious disease surveillance and public health response. Beyond COVID-19, public health, healthcare professionals, and scientists can use genomic surveillance to identify infectious disease clusters with greater speed and accuracy, and at earlier time points to help trigger and guide a more informed and effective public health response.

# [Authors/Acknowledgements/Affiliations](https://nextstrain.org/community/tgennorth/arizona-covid-19/)

- TGen
  - Dave Engelthaler
  - Jolene Bowers
  - Darrin Lemmer
  - Megan Folkerts
  - Chris French
  - Hayley Yaglom
  - Danielle Vasquez
  - Ashlyn Pfeiffer
  - Amber Jones
  - Adina Doyle
  - Krystal Sheridan
  - Sara Wilbur
  - Jim Schupp
  - Marjorie Nguyen

- NAU
  - Paul Keim
  - Crystal Hepp
  - Jason Ladner
  - Greg Caporaso
  - Jason Sahl
  - Evan Bolyen

- ASU
  - Efrem Lim
  - Brenda Hogue

- UA
  - Brendan Larsen
  - Michael Worobey

## Acknowledgements:

The ACGU would like to gratefully acknowledge the tireless efforts of Arizona’s local, tribal and state public health officials and healthcare workers, and to thank the other laboratories around the world submitting SARS-CoV-2 genomes to be shared with the scientific community.

The complete Genomic Epidemiology of SARS-CoV-2 in Arizona Narrative is located [here](https://nextstrain.org/community/narratives/tgennorth/arizona-covid-19).

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-tgen.png" width="220">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-nau.png" width="220">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-ua.png" width="220">
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-asu.png" width="220">
```
