---
title: "AZ-Strain: Genomic Epidemiology of SARS-CoV-2 in Arizona"
authorLinks: "https://www.tgen.org/patients/pathogen-microbiome/"
affiliations: "ACGU, TGen"
date: "2020 Sep 14"
dataset: "https://nextstrain.org/community/tgennorth/arizona-covid-19/7-13-20?d=map"
abstract: "Maintained by the AZ COVID Genomics Union (ACGU).

The Arizona COVID-19 Genomics Union (ACGU) uses state-of-the-art sequencers and custom computational analysis workflows to sequence patients’ SARS-CoV-2 RNA genomes and to track the mutations in the virus to map its spread. By comparing mutations observed in Arizona to those present in strains circulating across the globe, we are determining when and from where the virus is coming into Arizona. By comparing strains within Arizona, the ACGU is also mapping the virus' transmission within and among Arizona’s communities.

From among the 240,000+ positive cases in Arizona, the ACGU has sequenced SARS-CoV-2 genomes from over 5,000 COVID-19 positive samples, making this sequencing effort one of the most extensive in the nation.

The ACGU is a collaboration of scientists from the Translational Genomics Research Institute (TGen), Northern Arizona University (NAU), University of Arizona (UArizona), and Arizona State University (ASU)."
---

# [Arizona’s COVID-19 Timeline](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

COVID-19 has hugely impacted Arizona over the past nine months, sweeping through urban and rural areas, tribal nations, and correctional and healthcare facilities.

Arizona detected its first COVID-19 case on January 26, but it was not until late March that state public health officials reported widespread community transmission. Throughout June and July, hospitalizations were at an all-time high, infection rates among individuals aged 20–44 skyrocketed, and Arizona’s case rate per 100,000 was almost double what was reported for the United States. As of October 27, 2020, Arizona has reported over 240,000 cases and 5,800 deaths.

The Arizona Department of Health Services maintains a [COVID-19 Data Dashboard](https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php), updated daily.

* [Arizona’s first COVID-19 cases were successfully contained](https://nextstrain.org/community/narratives/tgennorth/arizona-covid-19?n=3)
* [SARS-CoV-2 is introduced into Arizona](https://nextstrain.org/community/narratives/tgennorth/arizona-covid-19?n=4)
* [COVID-19 is declared a global pandemic by the WHO](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/interactive-timeline#event-71)
* [COVID-19 is declared “Widespread” in ARizona](https://nextstrain.org/community/narratives/tgennorth/arizona-covid-19?n=7)
* [ACGU published analysis of early pandemic in Arizona](https://mbio.asm.org/content/11/5/e02107-20)
* [Arizona’s latest genomic epidemiology story](https://nextstrain.org/community/narratives/tgennorth/arizona-covid-19?n=11)

```auspiceMainDisplayMarkdown
<img src="https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-timeline.png" width=75%>
```

# [Arizona SARS-CoV-2 Genomes in a Global Context](https://nextstrain.org/community/tgennorth/arizona-covid-19/?c=location&p=full)

COVID-19 came into Arizona from many different places, as demonstrated by this phylogenetic tree that displays the relatedness of different strains of the SARS-CoV-2 virus. The strains that are now circulating in Arizona are representative of strains from around the world; however, the vast majority of Arizona’s cases are from strain types shared with other U.S. states or European countries.

This build includes a dataset of 3010 SARS-CoV-2 genomes sequenced from COVID-19 positive patient specimens from across Arizona. Positive specimens came from the TGen North Clinical Laboratory, Arizona State Public Health Laboratory, and Sonora Quest Laboratory between March and August. The Arizona genomes are shown here with a set of 532 genomes from around the world shared in the [GISAID](https://www.gisaid.org/) database.

# [Page 4](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst?d=tree&f_division=Arizona&p=full&s=TG268188::Coconino::B.2::0.99::20-5463596::2020-03-17)

COVID-19 hit Arizona on January 22, 2020, when a 26-year-old male presented to a clinic in Maricopa County, reporting a two-day history of nonproductive cough. He denied fever and other symptoms at presentation. He had returned from travel to China (including Wuhan City) three days prior to presentation, and reported having contact with one person with cough and fever while in China. This case was rapidly quarantined. Despite his specimens testing positive at  the CDC eighteen days after diagnosis, no SARS-CoV-2 transmission was detected among ten high-risk contacts, including one intimate contact, based on serial SARS-CoV-2 testing. The CDC [published a genome sequence](https://www.ncbi.nlm.nih.gov/nuccore/MN997409.1) for the sample collected from this initial case. 
 
Genomic analysis by the ACGU supports the public health contact tracing conclusions: Continued genomic surveillance shows no evidence that this first case resulted in cryptic (undetected) transmission, as no strains of virus collected later in Arizona descended from this lineage. 
 
**_Rapid public health response efforts, including quarantining, testing, and contact tracing, prevented this case from potentially igniting a much earlier outbreak in Arizona_**

# [Evidence of Multiple Introductions of SARS-CoV-2 into Arizona in February and March, Mostly from Domestic Travel](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

# [SARS-CoV-2 was introduced into Arizona at least twelve times by early April](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Multiple distinct SARS-CoV-2 lineages co-circulated in Arizona; genomic analyses highlight eleven distinct lineages and/or sublineages in the state by early April. None of these lineages were unique to Arizona; they have all been documented in other parts of the United States and, in most cases, in countries around the world. In fact, the geographic ubiquity of the major SARS-CoV-2 lineages, along with the common observation of identical virus genomes sampled in multiple US states and in other countries and continents, clearly demonstrate how frequently this virus has been moved among locales. Given the timing and size of the Arizona outbreak in March, it is unlikely that, relative to outbreaks in other locations, any of these eleven lineages arose within Arizona. Therefore, the number of observed distinct lineages (twelve, including AZ1) represents a conservative estimate for the number of independent introductions of SARS-CoV-2 into Arizona.

# [SARS-CoV-2 was introduced into Arizona in mid to late February](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Molecular clock analyses showed the introductions of SARS-CoV-2 in Arizona that resulted in community transmission were in February and March. 
 
XX% of Arizona’s sequences descend from strains that were initially circulating widely in Europe but have since dominated the outbreak in the US. In addition, we show that the first reported case of community transmission in Arizona descended from the Washington state outbreak that was discovered in late February. Notably, none of the observed transmission clusters are epidemiologically linked to the original travel-related case in the state, suggesting successful early isolation and quarantine.

# [Introductions likely mostly resulted from domestic travel](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

The first reported case of “community” transmission occurred in Arizona in early March and descended from the Washington state outbreak that was discovered in February. More than 80% of the SARS-CoV-2 genome sequences from Arizona COVID-19 cases descended from at least eleven separate lineages that were initially circulating widely in Europe, and by travel have since dominated the outbreak throughout the US. None of the observed transmission clusters are epidemiologically linked to the original travel-related case in Arizona, suggesting successful early isolation and quarantine.

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-1.png)
```

# [COVID-19 Level in Arizona is “Widespread”](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

On March 26th, the Arizona Department of Health Services updated the community transmission level of COVID-19 in Arizona to “Widespread,” indicating that twelve or more of the fifteen counties throughout the state had confirmed COVID-19 cases. 
 
The Nextstrain dataset shows XXX  60 genomes from 11 counties reporting COVID-19 cases from up to March 26th. Colors indicate the strain of virus, indicating that multiple lineages spread among several counties within Arizona.

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-2.png)
```

# [An Early Pandemic Analysis of SARS-CoV-2 Population Structure and Dynamics in Arizona](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Initial findings reported by the ACGU suggest that the state experienced no new COVID-19 cases after the first reported case in late January. Thereafter, at least eleven distinct incursions occurred between mid-February and early April. The published results appear in the scientific journal mBio. This [study](https://tgen.us10.list-manage.com/track/click?u=6b0e128788a10661a1fb2d549&id=f3e6925358&e=81c3146932) was supported by: The NARBHA Institute, Flinn Foundation, The Virginia G. Piper Charitable Trust, Blue Cross and Blue Shield of Arizona, the National Institutes of Health, the David and Lucile Packard Foundation, the University of Arizona College of Science and Office of Research Innovation and Impact, and BIO5 Institute.

# [Outbreak of COVID-19 in a Linked Group Home Setting](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-3.png)
```

# [Outbreak Story 2](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
```

# [Outbreak Story 3](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Coming Soon

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/az-covid-4.png)
```

# [Rapid Transmission of SARS-CoV-2 in Northern Arizona Tribal Community with Low Mortality](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst?c=gt-S_245&m=div&p=full)

- Tribal introduction story / H245Y (T67I) and R209I mutations

Although introduction of SARS-CoV-2 in Arizona was [likely in February](https://mbio.asm.org/content/11/5/e02107-20), COVID-19 didn’t hit a mid-sized (population = 18K) Arizona Native American (NA) community until over a month later. This NA reservation has over 13,500 members living in nine main communities; the capital and largest community has a population of about 3,000 residents. The community’s first COVID-19 case was diagnosed on April 1; shortly thereafter, the area experienced the highest rate of infection documented to date (25% of the population on the reservation). As of August 2020, it is estimated that more than one-eighth of the residents have tested positive for COVID-19, with more than 2,200 reported cases. The majority of the testing has been conducted in the context of contact investigations instead of widespread community testing, which may contribute to the high positivity rate. However, with 34 deaths, the community has a case fatality rate of ~1.4%, one of the lowest recorded to date – a remarkable outcome, considering the high comorbidity burden and multi-generational living conditions typical of NA communities. The low fatality rates may be driven in large part by the successful on-ground rapid test and treat interventions from [local health workers](https://www.nejm.org/doi/full/10.1056/NEJMc2023540).
 
This tribe’s index case (sample collected March 29) was a 47-year old health care employee that had an onset of fever and cough on March 25, a history of recent travel to Phoenix, Arizona, but continued to work despite being reportedly symptomatic. Genomic analysis shows Patient A1 seeded the outbreak comprising the majority of cases on this reservation, and mostly exclusive to this reservation, illustrated by the monophyletic clade defined by an amino acid change in the spike protein, H245Y. All identified cases in April and most in May from this rural Arizona community make up this clade illustrating rapid spread of a single clone.

# [Summary](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Return to timeline - highlight uniqueness of Arizona epi circumstances - anti-seasonal effects as people congregate indoors in summer.
 
Our data tell us that the initial Arizona case with travel to China in January did not lead to sustained local transmission, and that multiple introductions (>12) of the virus occurred from multiple countries and states, some of which contributed to significant transmission in Arizona. Mutations have been found that may lead to increased transmission and decreased virulence.

# [Public Health Significance](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Using genomic epidemiology, we have a better understanding of the circulation of strains of the SARS-CoV-2 virus among Arizona communities, which is critical to guide an effective public health response to curb the spread of the virus now and in the future. Whole genome sequencing of the viral genome from infected persons clinical specimens, along with information about the collection of the specimens (when and where), allow for phylogenetic (i.e., family tree) reconstruction to track the circulation and transmission of the virus locally and globally. We use these data to understand viral movement, identify origins of introductions of different COVID-19 strains to the region, locate hot spots of local transmission and clusters of disease, and possibly determine the source of infection. We can also study mutations in the viral genome and their potential for causing increased transmission or virulence.

# [Methodology](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

Samples positive for Sars-CoV-2 (confirmed by RT-PCR) were extracted using the MagMax Viral Pathogen II kit from ThermoFisher. Extracted RNA was prepared for either whole meta-transcriptome sequencing or tiled amplicon sequencing. Whole meta-transcriptome sequencing was performed using either Tecan’s Ovation SoLo RNA Seq kit, or TakaraBio’s SMART-Seq Stranded kit, both according to manufacturer’s specifications. Tiled amplicon sequencing was performed using the ARTIC protocol (v1) by Josh Quick, and adapter addition was done using Seqwell’s plexWell kit, or Illumina’s DNA-prep kit, both according to manufacturer’s instructions for small amplicons. Sequencing was done on a NextSeq 550 with v2.5 chemistry, or a MiSeq with V3 chemistry.

# [Authors/Acknowledgements/Affiliations](https://nextstrain.org/community/tgennorth/arizona-covid-19/gst)

- TGen
  - Dave Engelthaler
  - Jolene Bowers
  - Darrin Lemmer
  - Megan Folkerts
  - Chris French
  - Hayley Yaglom
  - Danielle Vasquez
  - Ashlyn Pfeiffer
  - Amber Jones
  - Adina Doyle
  - Krystal Sheridan
  - Sara Wilbur

```auspiceMainDisplayMarkdown
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-tgen.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-nau.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-ua.png)
![images](https://pathogen.tgen.org/epitools/static/nextstrain/images/logo-asu.png)
```


